test/libsolidity/SolidityNatspecJSON.cpp

```diff
@@ -355,6 +355,51 @@ BOOST_AUTO_TEST_CASE(private_state_variable)
 	checkNatspec(sourceCode, "test", userDoc, true);
 }
 
+BOOST_AUTO_TEST_CASE(event)
+{
+	char const* sourceCode = R"(
+		contract ERC20 {
+			/// @notice This event is emitted when a transfer occurs.
+			/// @param from The source account.
+			/// @param to The destination account.
+			/// @param amount The amount.
+			/// @dev A test case!
+			event Transfer(address indexed from, address indexed to, uint amount);
```

Comment:
So we just ignore that it is indexed?

Reply:
Yes. The "indexed" will be present in the ABI, and tools can get it from there.